docs/sources/guides/whats-new-in-v5-1.md

```diff
@@ -0,0 +1,111 @@
+More information in the [Provisioning documentation](/features/datasources/prometheus/).
+
+<div class="clearfix"></div>
+
+## New template variable interpolation syntax
+
+
+## Alerting
+
+New enhancements includes support for retries on failed alert queries and updating correct state when pausing/unpausing alerts.
+
+## Datasources
+
+### Prometheus
+
+New enhancements includes support for Prometheus histograms in the heatmap panel and autocomplete of template variables query editor. More information in the [Prometheus data source documentation](/features/datasources/prometheus/).
+
+### Cloudwatch
+
+New enhancements includes support for high resolution metrics and dimension filtering. More information in the [Cloudwatch data source documentation](/features/datasources/cloudwatch/).
+
+### PostgreSQL
+
+New enhancement includes support for filling in values for missing intervals and better precision handling and data type support for time columns. More information in the [PostgreSQL data source documentation](/features/datasources/postgres/#time-series-queries).
+
+### MySQL
+
+New enhancements includes support for filling in values for missing intervals, better precision handling and data type support for time columns and any column except time and metric is now treated as a value column. More information in the [MySQL data source documentation](/features/datasources/mysql/#time-series-queries).
+
+### Microsoft SQL Server
+
+{{< docs-imagebox img="/img/docs/v51/mssql_query_editor_showcase.png"  max-width= "800px" class="docs-image--right" >}}
+
+Grafana v5.1 now ships with a built-in Microsoft SQL Server (MSSQL) data source plugin that allows you to query and visualize data from any Microsoft SQL Server 2005 or newer, including Microsoft Azure SQL Database. Have logs or metric data in MSSQL? You can now visualize that data and
+define alert rules on it like any of our other data sources.
+
+Same enhancements as described for PostgreSQL and MySQL are included in the MSSQL datasource.
+
+Please read [Using Microsoft SQL Server in Grafana documentation](/features/datasources/mssql/) for more detailed information on how to get started and use it.
+
+<div class="clearfix"></div>
+
+## Panels
+
+### Graph Panel
+
+New enhancements includes support for multiple series stacking in histogram mode, thresholds for right Y axis, aligning left and right Y-axes to one level and additional units. More information in the [Graph panel documentation](/features/panels/graph/).
+
+### Table Panel
+
+New enhancements includes support for mapping a numeric value/range to text and additional units. More information in the [Table panel documentation](/features/panels/table_panel/#string).
+
+### Singlestat Panel
+
+New enhancements includes support for adding color to prefix and postfix and additional units. More information in the [Singlestat panel documentation](/features/panels/singlestat/).
+
+## UX improvements
```

Comment:
We can remove this. 

In general, I only think UX changes should be mentioned if they change a flow or is very big.